src/receivers/HTTPReceiver.ts

```diff
@@ -506,15 +519,26 @@ export default class HTTPReceiver implements Receiver {
   private handleInstallRedirectRequest(req: IncomingMessage, res: ServerResponse) {
     // This function is only called from within unboundRequestListener after checking that installer is defined, and
     // when installer is defined then installCallbackOptions is always defined too.
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    const [installer, installCallbackOptions] = [this.installer!, this.installCallbackOptions!];
-
-    installer.handleCallback(req, res, installCallbackOptions).catch((err) => {
+    /* eslint-disable @typescript-eslint/no-non-null-assertion */
+    const [installer, installCallbackOptions, installUrlOptions] = [
+      this.installer!,
+      this.installCallbackOptions!,
+      this.installUrlOptions!,
+    ];
+    /* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
I think we want to turn the eslint rule back on here? If so, this should be an `eslint-enable` directive instead:

```suggestion
    /* eslint-enable @typescript-eslint/no-non-null-assertion */
```